shaky/src/main/java/com/linkedin/android/shaky/ShakeDelegate.java

```diff
@@ -25,6 +25,10 @@
  */
 public abstract class ShakeDelegate {
 
+    public static final int SENSITIVITY_LIGHT  = 22;
```

Comment:
Can we make an `@IntDef` for these so lint will check the ints that get returned from `getSensitivityLevel()`?